### PR TITLE
Merge 1.3 => 1.4

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Datasource/ResultRecord/MongoDbOdm/Product/ValuesTransformer.php [hunk 1]
```diff
@@ -78,13 +79,20 @@ protected function prepareDateData(array $result, array $attribute)
      *
      * @return array
      */
-    protected function prepareMediaData(array $result, array $attribute)
+    protected function prepareMediaData(array $result, array $attribute, $locale, $scope)
```
Comment (solivier) [inline src/Pim/Bundle/DataGridBundle/Datasource/ResultRecord/MongoDbOdm/Product/ValuesTransformer.php]: missing doc

Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Datasource/ResultRecord/MongoDbOdm/Product/ValuesTransformer.php]: update phpdoc please

**Comment by solivier** [comment]
gtm after com' fixed

**Comment by skeleton** [comment]
gtm after fixed

**Comment by juliensnz** [comment]
GTM

File: src/Pim/Bundle/DataGridBundle/Datasource/ResultRecord/MongoDbOdm/Product/ValuesTransformer.php (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/DataGridBundle/Datasource/ResultRecord/MongoDbOdm/Product/ValuesTransformer.php]: done, thanks!

**Pull request merged**